### ISSUE TITLE
Plugin Requests

### ISSUE THREAD

**Issue by Lagomorphius** [Tharki-God/BetterDiscordPlugins#73]
Can you please make an Unfriend and Guild Ban/Kick notifier? The one I had is not managed, the owner only updates when it has pull requests so keeping it up to date is unstable and unreliable.

Could you also make a message purger for DMs and servers so you can delete your messages from a specific server or person?

**Comment by Tharki-God** [comment]
I can make the message purger if you can pay me :)

**Comment by Lagomorphius** [comment]
Would you maintain it if it breaks? What about the unfriend and server notifier?

**Comment by Tharki-God** [comment]
> Would you maintain it if it breaks? What about the unfriend and server notifier?

Yes I maintain all my plugins. And about notifer plugin I will think about it when I get back home after 2 days

**Comment by iGetUploaded** [comment]
> 

What a great idea miss the old one that doesn't get any updates anymore

**Comment by Lagomorphius** [comment]
I notice my friends count going down sometimes and it bothers me not knowing who did it.

**Comment by Tharki-God** [comment]
Pay me and I will make it

**Comment by Lagomorphius** [comment]
How much for both plugins

**Comment by Tharki-God** [comment]
1k INR for each

**Comment by Lagomorphius** [comment]
So 12.50 USD basically for each? I'd feel bad paying that low, how about 35 USD

**Comment by Tharki-God** [comment]
> So 12.50 USD basically for each? I'd feel bad paying that low, how about 35 USD

Sure. I will share details for payment later

**Comment by Lagomorphius** [comment]
I have a big list of things I gotta buy first as well as pay off some artists I owe money too

**Comment by Tharki-God** [comment]
> I have a big list of things I gotta buy first as well as pay off some artists I owe money too

Yeah do that and add me on discord I will let you know when I can make the plugins and other details there only

**Comment by RobinRMC** [comment]
@Lagomorpheus I want to warn you that a message purger has a very high chance of getting your account terminated. Deleting a lot of messages spams Discord's API and that could trigger the selfbot detection and get your account disabled.

The removal notifications are something I'm missing too. I also used GuildAndFriendRemovalAlerts and it was a great way to get notified of what events happen that apply to you.

**Comment by RobinRMC** [comment]
Group removal notifications would be a good addition to the removal notifications plugin, so if the plugin is going to be written, consider adding this feature.

**Comment by Lagomorpheus** [comment]
I'm aware of the risks of automatic message purging, but as long as it remains with API limits it will be fine, there was a plugin called MessageCleaner that did it safely before the developer gave up due to code changes and the browser based one called undiscord is still up.

**Comment by Zzeyy** [comment]
Man's so rich that he basically offers almost three times the money that's been asked LOL

**Comment by RDP8** [comment]
What's the status of the Guild and Friend removal alerts plugin? Is it being worked on?

**Comment by RDP8** [comment]
> Can you please make an Unfriend and Guild Ban/Kick notifier? The one I had is not managed, the owner only updates when it has pull requests so keeping it up to date is unstable and unreliable.
> 
> Could you also make a message purger for DMs and servers so you can delete your messages from a specific server or person?

https://github.com/iyu46/RemovedConnectionAlerts

**Comment by Tharki-God** [comment]
There goes 30$
